test/extended/networking/multinetpolicy.go

```diff
@@ -33,19 +28,6 @@ var _ = g.Describe("[sig-network][Feature:MultiNetworkPolicy][Serial][apigroup:o
 	oc := exutil.NewCLIWithPodSecurityLevel("multinetpol-e2e", admissionapi.LevelBaseline)
 	f := oc.KubeFramework()
 
-	// Ensure that MultiNetPolicy stays in the same enabled/disabled state
-	// after the test, that it was in before the test
-	var previousUseMultiNetworkPolicy bool
-	g.BeforeEach(func() {
-		previousUseMultiNetworkPolicy = isMultinetNetworkPolicyEnabled(oc)
-	})
-
-	g.AfterEach(func() {
-		if !previousUseMultiNetworkPolicy {
```

Comment:
`previousUseMultiNetworkPolicy` should be true since https://github.com/openshift/release/pull/41872 merged.. but I see CI failing, retest the job.. let's see...